### PR TITLE
feat: frontend rewrite to sveltekit with basic features

### PR DESCRIPTION
**Pull request by AyushSehrawat**
- Rewrite of frontend to sveltekit
- Added /user endpoint to api
- `uvicorn.run("main:app", host="localhost", port=8080, reload=False)` Uvicorn reload run is false in code, in future based on environment, prod or dev we can make it dynamic
- commented out `self.state = MediaItemState.UNKNOWN` from media.py

### CODE REVIEW

File: backend/controllers/default.py [hunk 1]
```diff
@@ -11,3 +14,15 @@ async def root(request: Request):
         "success": True,
         "message": "Iceburg is running!",
     }
+
+
+@router.get("/user")
+async def get_rd_user():
+    api_key = settings_manager.get("realdebrid")["api_key"]
```
Comment (Gaisberg) [inline backend/controllers/default.py:21]: this should be in realdebrid.py

Comment (dreulavelle) [reply backend/controllers/default.py:21]: Did we want to keep all endpoints in controllers dir?

Comment (AyushSehrawat) [reply backend/controllers/default.py]: > Did we want to keep all endpoints in controllers dir?

Yes, consistency will help in finding stuff 😄

**Review by dreulavelle — APPROVED** [review]
Hope it works!

**Pull request merged**